test/test_manifest.json

```diff
@@ -6264,5 +6264,12 @@
          "value": "PDF.js PDF.js PDF.js"
         }
       }
+   },
+   { "id": "issue14562",
+      "file": "pdfs/issue14562.pdf",
+      "md5": "565b0a1e46a32e907837506a10d25277",
+      "rounds": 1,
+      "link": true,
+      "type": "text"
```

Comment:
Given that none of the changes in this patch show up in a `text` test, let's save time/resources by making this `"type": "other"` instead.